### ISSUE TITLE
[7.0.3] Warning message when verify the CVE scanner

### ISSUE THREAD

**Issue by tuxmaster** [greenbone/gvm#65]
When the manger verify the CVE scanner, than it will log an SQL warning.
1. call openvasmd --get-scanners:
08b69003-5fc2-4037-a479-93b440211c73  OpenVAS Default
6acd0832-df90-11e4-b9d5-28d24461215b  CVE
2. call openvasmd --verify-scanner 6acd0832-df90-11e4-b9d5-28d24461215b:
Scanner version: OTP/2.0.

And at the openvasmd.log file an warning is logged:
md manage:WARNING:2018-04-09 08h53.49 utc:6978: sql_close: attempt to close db with open statement(s)

**Comment by tuxmaster** [comment]
But the scanner and the manager are running on the same server and communicate via unix sockets.
Where here are the ssl problem?

**Comment by tuxmaster** [comment]
The result is the same.
I can call the scanner verifying via gsa or on the command line.

**Comment by cfi-gb** [comment]
@tuxmaster I don't think that the two previous posts of that user has any meaning for your issue. This seems to be just some randomly copy'n'paste text (maybe by a bot?) which is more or less just off-topic.

**Comment by cfi-gb** [comment]
@tuxmaster Looks to me that you could also close the other duplicated issue for 7.0.2: https://github.com/greenbone/gvm/issues/19

**Comment by timopollmeier** [comment]
This looks like a duplicate of #19 to me as well.
